cognite_toolkit/cdf_tk/load.py

```diff
@@ -412,18 +424,23 @@ def load_file(self, filepath: Path, ToolGlobals: CDFToolConfig) -> Group:
         for capability in raw.get("capabilities", []):
             for _, values in capability.items():
                 if len(values.get("scope", {}).get("datasetScope", {}).get("ids", [])) > 0:
-                    values["scope"]["datasetScope"]["ids"] = [
-                        ToolGlobals.verify_dataset(ext_id)
-                        for ext_id in values.get("scope", {}).get("datasetScope", {}).get("ids", [])
-                    ]
+                    if self.mode != "all_only":
+                        values["scope"]["datasetScope"]["ids"] = [
+                            ToolGlobals.verify_dataset(ext_id)
+                            for ext_id in values.get("scope", {}).get("datasetScope", {}).get("ids", [])
+                        ]
+                    else:
+                        # If we are running a clean, this is a no-op, but it needs to be valid
+                        values["scope"]["all"] = {}
+                        values["scope"].pop("datasetScope")
         return Group.load(raw)
```

Comment:
We need to handle empty dict here now, as if we skip scopes -> we need to skip capabilities -> we may need to skip the group (in case it only has dataSetScope capabilities.

Reply:
We are not skipping scopes, we are replacing datasetscopes with scope all. The whole point is to avoid that parsing of group with datasetscopes fail if datasets don't exist in the CDF project, but we need all the groups as we don't know which ones exist, only that the groups with datasetscopes where the dataset does not exist, will not exist themselves.